src/components/Navbar.tsx

```diff
@@ -7,21 +7,21 @@ export default function Navbar() {
   // this state is used for autocomplete.
   const [input, setInput] = useState("");
 
-  return (
-    <div className="navbar top-0 flex h-20 flex-row border-b-2 bg-white">
-      {/* untested on mobile ^ got rid of sticky because leaflet map on search hid the search results from the dropdown in here. navbar will not stay at the top, PLEASE FIX SOMEONE!! */}
+  return (<>
+    <div className="navbar flex top-0 h-20 flex-row border-b-2 bg-white absolute">
       <div className="navbar-start w-fit sm:w-1/2 mr-5 sm:mr-0">
         <Link href="/Landing" className="btn btn-ghost my-auto h-16">
```

Comment:
please use the next/image element so we can avoid having to disable eslint (i can do this if needed)

Reply:
can you replace the img with Images by 8pm 11/16? otherwise i will leave it as is.